server/routes/visit/visitEventsTimelineBuilder.ts

```diff
@@ -0,0 +1,82 @@
+import eventAuditTypes from '../../constants/eventAudit'
+import { notificationTypes } from '../../constants/notifications'
+import { requestMethodDescriptions } from '../../constants/requestMethods'
+import { EventAudit, VisitBookingDetailsDto } from '../../data/orchestrationApiTypes'
+
+export type MojTimelineItem = {
+  label: { text: string }
+  text: string
+  datetime: {
+    timestamp: string
+    type: 'datetime'
+  }
+  byline?: { text: string }
+  attributes: { 'data-test': string }
+}
+
+export default ({
+  events,
+  visitNotes,
+}: {
+  events: EventAudit[]
+  visitNotes: VisitBookingDetailsDto['visitNotes']
+}): MojTimelineItem[] => {
+  // Filter to only supported event types and reverse order for display
+  const filteredEvents = events.filter(event => event.type in eventAuditTypes).reverse()
+
+  return filteredEvents.map((event, index) => {
```

Comment:
Can't you just drop the `filter().reverse()` in here then there's no need to assign a new var? So `events.filter().reverse().map(....` ?

Reply:
Good suggestion. I've done this and it looks nicer now 🙂